### ISSUE TITLE
How to use it like RazorEngine?

### ISSUE THREAD

**Issue by bnuzhouwei** [elfalem/NVue#1]
```csharp
using RazorEngine;
using RazorEngine.Templating;
namespace RazorDemo.ConsoleDemo
{
    class Program
    {
        static void Main(string[] args)
        {
            var template = "Hello @Model.Name, welcome to use RazorEngine!";
            var result = Engine.Razor.RunCompile(template, "templateKey1", null, new { Name = "World" });
            Console.WriteLine(result);
            Console.Read();
        }
    }
}
```

**Comment by elfalem** [comment]
I've just implemented this and released a new version (v0.2.0).

It can be used as follows:
````csharp
var template = "<template>Hello {{Name}}, welcome to NVue!</template>";
var result = NVueTemplateEngine.RunCompile(template, "testtemplate", null, new { Name = "World" });
Console.WriteLine(result); // Hello World, welcome to NVue!
````

**Comment by bnuzhouwei** [comment]
Nice work!
